### PR TITLE
misleading Domain description fixed

### LINKED ISSUE #12860: Bug: misleading `Domain` description
### What are you trying to achieve?

As pointed out in this discussion in the docs https://github.com/saleor/saleor-docs/discussions/761 the `Domain` description is misleading.

### Steps to reproduce the problem

https://docs.saleor.io/docs/3.x/api-reference/miscellaneous/objects/domain

### What did you expect to happen?

Update the description - state it's the API domain

### Logs

_No response_

### Environment

Saleor version: …
OS and version: …

### PR DESCRIPTION
**Pull request by nayeem01**
I want to merge this change because it Resolves #12860

# Impact

- [x] misleading Domain description fixed

### CODE REVIEW

File: saleor/graphql/shop/types.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ class Domain(graphene.ObjectType):
     url = graphene.String(description="Shop's absolute URL.", required=True)
```
Comment (maarcingebala) [inline saleor/graphql/shop/types.py:68]: We could also change this value: "The absolute URL of the API".

Comment (nayeem01) [reply saleor/graphql/shop/types.py]: @maarcingebala done

Comment (maarcingebala) [reply saleor/graphql/shop/types.py]: Looks good, but we need to also generate an updated schema.graphql with `python manage.py get_graphql_schema > saleor/graphql/schema.graphql`.

**Comment by IKarbowiak** [comment]
@nayeem01 please regenerate the `schema`. You can do this by calling `python manage.py get_graphql_schema > saleor/graphql/schema.graphql`

**Comment by aniav** [comment]
Hey @nayeem01 do you intend to finish off this PR? 🙏

**Comment by nayeem01** [comment]
> Hey @nayeem01 do you intend to finish off this PR? 🙏 

Yes. I will try to work on it next week.

**Review by maarcingebala — APPROVED** [review]

**Comment by maarcingebala** [comment]
@nayeem01 Thanks for looking into this, there is still one change that needs to be reflected in the `schema.graphql` file. There are two changed descriptions in the code, but only one is included in the schema now.

**Review by fowczarek — APPROVED** [review]

**Comment by nayeem01** [comment]
@maarcingebala please check

**Review by IKarbowiak — APPROVED** [review]

**Comment by maarcingebala** [comment]
Looks good, thank you for your contribution to Saleor @nayeem01!

**Pull request merged**